### PR TITLE
Individual Assignment 5

### PR DESCRIPTION
**Pull request by AlvaroAlonso-0**

### CODE REVIEW

File: Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    --return 'count()'
+""")
+
+# Retrieve at least two members of Schwarzenegger's political party. Make sure only persons are returned
+# SPARQL:
+"""
+SELECT ?partyName WHERE{
+  wd:Q2685 wdt:P102 ?poliParty .
+  ?member wdt:P102 ?poliParty FILTER (?member != wd:Q2685) FILTER NOT EXISTS {?member wdt:P570 []} .
+  ?member rdfs:label ?partyName FILTER (lang(?partyName) = "es") .
```
Comment (dgarijo) [inline Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py:167]: here you are not ensuring the members are persons

File: Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+} LIMIT 3
+"""
+
+# Kypher:
+kgtk("""
+    query -i all
+    --match '(:Q2685)-[:P102]->(poliparty)<-[:P102]-(member)-[:label]->(partyName)'
```
Comment (dgarijo) [inline Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py]: The requested query is for members, not which parties they belong to

File: Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+# node1	node2	label
+# Q2685	Q22686	path
+
+kgtk("""
+    add-labels -i $TEMP/path-query.tsv
+""")
+
+# Calculate all the paths between Trump and Schwarzenegger (max hops: 3)  
+kgtk("""
+    #TO DO
```
Comment (dgarijo) [inline Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py]: missing!

File: Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+kgtk("""
+    reachable-nodes -i all
+      --root Q2685
+      --props P40 P22 P25 P3373 P26
+      --label Schwarzenegger_family
+""")
+
+# What are the 10 most relevant actors (pagerank) in the graph? (Use graph-statistics command to calculate page rank, and then filter only actors)
+kgtk("""
+    #TO DO
```
Comment (dgarijo) [inline Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py]: missing!

**Review by dgarijo — CHANGES_REQUESTED** [review]
nicely done. You have left out a couple of exercises. Can you pls complete them?

File: Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py (hunk 2 above)
Comment (AlvaroAlonso-0) [reply Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py]: The query returns the members, maybe the name leads to a misunderstanding

Comment (dgarijo) [reply Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py]: yep, you are right, dismiss this comment

Comment (dgarijo) [inline Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py]: You are also not ensuring the party members are persons here

File: Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py (hunk 1 above)
Comment (AlvaroAlonso-0) [reply Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py:167]: Just pushed the changes requested

Comment (dgarijo) [reply Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py:167]: This is not yet done. You should have ?member wdt:P31 wd:Q5 somewhere...

Comment (AlvaroAlonso-0) [reply Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py:167]: Yes, I forgot to save the last change before the commit. Now should be correct

**Pull request merged**